src/components/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -106,3 +107,16 @@ WithChecklist.args = {
     value: JSON.stringify(checkboxValue),
 };
 WithChecklist.argTypes = { value: { type: "string" } };
+
+export const WithCustomControls: Story<RichTextEditorProps> = (args: RichTextEditorProps) => (
+    <RichTextEditorComponent {...args} />
+);
```

Comment:
You could create a RichTextEditorTemplate and bind it instead of creating a new component (also for the other stories)
```suggestion
const RichTextEditorTemplate: Story<RichTextEditorProps> = (args: RichTextEditorProps) => (
    <RichTextEditorComponent {...args} />
);

export const WithCustomControls = RTETemplate.bind({});
```

Reply:
Changed